public/js/actions/index.js

```diff
@@ -0,0 +1,25 @@
+import fetch from 'unfetch';
```

Comment:
It's called... `fetch` :/ https://github.com/developit/unfetch/blob/master/src/index.js https://github.com/developit/unfetch#usage